### DISCUSSION TITLE
Welcome to Pmetrics Discussions!

### DISCUSSION THREAD

**Discussion by mnneely** [LAPKB/Pmetrics#59] [Announcements]
## 👋 Welcome!
  Because Pmetrics is now hosted on Github, for a more integrated experience, we have migrated our Forum to Github Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by sungulatiR** [comment]
Hi, I am SG working in a drug development research organization, have some knowledge of R and making an effort to delve in Pharmacokinetics model thru Pmetrics.

**Reply by Siel** [reply]
Welcome, Pmetrics is a ever improving package. We are very receptive to comments and possible optimizations. Please let us know any issue you find and we will do our best to help!

**Comment by sumithmathew9** [comment]
Thank you, Michael and team. I am Sumith. Not a pure PK scientist, neither a clinician, nor a pure analytical chemist, but a mix of all the three. I have lot of doubts on the philosophy of Pmetrics and Parametric methods. I request this community to help me learn :)

**Reply by mnneely** [reply]
We all learn together and we welcome contributors!
